packages/elements/src/tool-approval.tsx

```diff
@@ -0,0 +1,168 @@
+"use client";
```

Comment:
@haydenbleasel this is a lot of code for what is a really simple component, but I tried to follow a similar pattern like we already have in the remaining components such as the `plan.tsx` component.

Keen to hear what you would suggest could work best? Thanks